src/config/app.ts

```diff
@@ -143,24 +143,30 @@ const getRateLimits = (config): RateLimitOptions => {
    * Returns a subset of the required parameters for the
    * 'rate-limiter-flexible.RateLimiterRedis' object.
    */
-  return {
+  const obj: RateLimitOptions = {
```

Comment:
I know is a tmp variable but maybe `rateLimitOptions` or just `opts` is better than `obj`